pages/_app.jsx

```diff
@@ -1,6 +1,6 @@
 import { DefaultSeo } from 'next-seo'
 import Layout from '@components/Layout'
-import '@styles/globals.scss'
+import '@styles/main.scss'
```

Comment:
Do we need to change this name? this will cause some conflicts, If theres a reason for it we can work around it.